template.tpl

```diff
@@ -547,22 +917,14 @@ const getFromPath = (path, obj) => {
  * Each of its rows contains a priority and a *common event* path to look for.
  */
 const locate = (locator, obj, postFunction) => {
-  const ordLoc = locator.sort((x, y) => {
-    const xPriority = makeInteger(x.priority);
-    const yPriority = makeInteger(y.priority);
-    return xPriority > yPriority ? -1 : 1;
-  });
+  const ordLoc = locator.sort((x, y) => makeInteger(x.priority) > makeInteger(y.priority) ? -1 : 1);
```

Comment:
Yes, this is indeed the case: here it does not matter because there is a field requirement for the priority values to be unique so that they cannot be equal.